instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/__init__.py

```diff
@@ -75,7 +74,7 @@ def _traced_execute_command(func, instance, args, kwargs):
     tracer = getattr(redis, "_opentelemetry_tracer")
     query = _format_command_args(args)
     with tracer.start_as_current_span(
-        _CMD, kind=trace.SpanKind.CLIENT
+        args[0], kind=trace.SpanKind.CLIENT
     ) as span:
```

Comment:
Fallback `db.name` for span name.

Reply:
After looking at the implementation of client library [here](https://github.com/andymccurdy/redis-py/blob/master/redis/client.py#L935) I thought we can safely use it without having to worry about fallback. Do you think we need this?

Reply:
maybe the one concern is that redis changes their argument format in a backwards-incompatible way. But I agree that's quite unlikely as it would break all consumers.

Reply:
I think it is less likely for other instrumentations as well, I just want to be consistent because there may be some exporters that rely on the existence of `db.name` so we should provide it if we can.

Reply:
`db.name` will be set for redis instrumentation. From official redis docs `If none of these options are specified, the default db=0 is used`. This will be numeral value. While we all agree that it unlikely redis changes their format, but just in case if it changes what do guys think setting this db numeral as span name?

Reply:
I think it is fine.